src/main/java/org/example/internetshop/controllers/IndexController.java

```diff
@@ -0,0 +1,14 @@
+package org.example.internetshop.controllers;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class IndexController extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
```

Comment:
please configure Travis CI in your feature PRs

For now, please run mvn clean package and check your code style